cloudflare_worker/worker/karma.conf.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = function(config) {
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine'],
+    files: [
+      'dist/**/*.test.js'
+    ],
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: false,
+    browsers: ['ChromeHeadless'],
```

Comment:
Why is this line necessary? Do we have code that uses the DOM? I would think some sort of node test runner would be sufficient (and faster).

Reply:
The [worker runtime](https://developers.cloudflare.com/workers/runtime-apis/web-standards#web-global-apis) is similar to WorkerGlobalScope in browser, which provides some global functions not available in nodejs.